DasharoBootPolicies/BootPolicies.c

```diff
@@ -38,6 +40,7 @@ InitializeBootPolicies (
   EFI_STATUS  Status = EFI_SUCCESS;
   BOOLEAN *EfiVar;
```

Comment:
Please change the type of this variable if you want it to have more than 2 possible values. While this works for [edk2's idea of BOOLEAN](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/X64/ProcessorBind.h#L163), it may not be portable to other implementations, or may break if this implementation is changed. Values other than 1 or 0 are undefined according to UEFI specification, this may also be misleading for anyone reading the code.

Reply:
I see, but continuing with that logic i can't just change this to `UINT8` since it might break `BOOLEANS`, hence why I just added another variable and changed the rest of code accordingly.